src/graph/executor/query/TraverseExecutor.cpp

```diff
@@ -62,8 +62,12 @@ Status TraverseExecutor::buildRequestVids() {
     auto vidType = SchemaUtil::propTypeToValueType(metaVidType.get_type());
     for (; iter->valid(); iter->next()) {
       const auto& vid = src->eval(ctx(iter));
-      DCHECK_EQ(vid.type(), vidType)
```

Comment:
Why delete this?

Reply:
crash

Reply:
Should check why vid type is unexpected.

Reply:
ACK.